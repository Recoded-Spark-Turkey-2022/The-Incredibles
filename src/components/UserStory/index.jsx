import React from 'react';
import Button from '../UI/Button';
import story from '../../assets/pics/aboutpage/story.jpg';
import { useAuthState } from 'react-firebase-hooks/auth';
import { auth } from '../../firebase/firebase';
import Container from '../UI/Container';
import { useTranslation } from 'react-i18next';

function CallToAction() {
  const [t] = useTranslation();
  const [user] = useAuthState(auth);
  return (
    <Container>
      <div className="">
        <img src={story} className=" m-auto " />
      </div>
      <div className="text-center p-10 max-md:px-0 max-md:py-5">
        <h4 className="font-normal text-3xl py-5 text-cyan-600">
        {t('about.story.h4')}
        </h4>
        <div className="text-center  ">
          <p className="text-zinc-400	font-light  sm:text-xl text-base ">
          {t('about.story.p')}
          </p>
        </div>
      </div>
      <div className={user ? 'hidden' : 'text-center'}>
        <Button name="Sign Up" path="/signup" />
      </div>
    </Container>
  );
}

export default CallToAction;
