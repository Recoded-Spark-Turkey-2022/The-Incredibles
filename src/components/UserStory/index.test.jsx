import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import UserStory from '.';

describe('UserStory', () =>
  it('renders correctly', () => {
    const domTree = renderer
      .create(
        <Router>
          <UserStory />
        </Router>
      )
      .toJSON();
    expect(domTree).toMatchSnapshot();
  }));
