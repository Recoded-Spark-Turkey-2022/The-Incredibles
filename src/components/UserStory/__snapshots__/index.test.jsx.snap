// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserStory renders correctly 1`] = `
<div
  className="container max-w-screen-xl m-auto p-10 max-md:px-5"
>
  <div
    className=""
  >
    <img
      className=" m-auto "
      src="story.jpg"
    />
  </div>
  <div
    className="text-center p-10 max-md:px-0 max-md:py-5"
  >
    <h4
      className="font-normal text-3xl py-5 text-cyan-600"
    >
      about.story.h4
    </h4>
    <div
      className="text-center  "
    >
      <p
        className="text-zinc-400	font-light  sm:text-xl text-base "
      >
        about.story.p
      </p>
    </div>
  </div>
  <div
    className="text-center"
  >
    <button
      className="px-10 py-2.5 bg-cyan-600 text-white font-medium text-l leading-tight rounded-full shadow-md ease-in duration-300 hover:bg-purple-700 hover:shadow-lg hover:scale-110"
      onClick={[Function]}
      type="button"
    >
      Sign Up
    </button>
  </div>
</div>
`;
